### PR TITLE
MultiStation Expand backend testing

### PR DESCRIPTION
**Pull request by carolinemodic**
## Overview
Adds to the backend test suites for multi-station adjudication covering very edge cases and gaps more explictly.

### CODE REVIEW

File: apps/admin/backend/src/app.adjudication.test.ts [hunk 1]
```diff
@@ -772,6 +796,38 @@ test('adjudicateCvr requires active claim', async () => {
...
+  const queue = await apiClient.getBallotAdjudicationQueue();
+  const cvrId = assertDefined(queue[0]);
+
+  // The host claims the ballot, then adjudication succeeds
+  (await apiClient.claimAndLoadBallot({ cvrId })).unsafeUnwrap();
+  expect(await apiClient.adjudicateCvr({ cvrId, contests: [] })).toEqual(ok());
+
+  // The claim is now completed, not active — but re-adjudicating an
+  // already-adjudicated ballot is allowed without a fresh claim, which is
+  // how the host edits a previously adjudicated ballot
```
Comment (nikhilb4a) [inline apps/admin/backend/src/app.adjudication.test.ts:828]: I'm not sure I'm grasping why we want to allow adjudicating a ballot without an active claim

**Review by nikhilb4a — APPROVED** [review]
